test/neo4clj/test_utils.clj

```diff
@@ -0,0 +1,30 @@
+(ns neo4clj.test-utils
+  "Neo4j 4.2.5 does not work with JDK version 16 - ServiceBuilder cannot \"build\" and throws exception.
+
+   https://github.com/neo4j/neo4j-documentation/blob/4.2/embedded-examples/src/main/java/org/neo4j/examples/EmbeddedNeo4j.java"
```

Comment:
Why is this link relevant? Either supply a line about what value it holds or delete it.